Header header

float64 long_accel
float64 lcc_steer
bool acc_active
bool aeb_active
bool lcc_active
bool acc_fault
bool aeb_fault
bool lcc_fault
bool front_radar_fault
bool left_radar_fault
bool right_radar_fault
bool mobileye_fault
uint64 alive_rolling_counter